common/src/main/java/com/wynntils/features/user/overlays/MiniMapOverlayFeature.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.BufferBuilder;
+import com.mojang.blaze3d.vertex.BufferUploader;
+import com.mojang.blaze3d.vertex.DefaultVertexFormat;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.Tesselator;
+import com.mojang.blaze3d.vertex.VertexFormat;
+import com.mojang.math.Matrix4f;
+import com.mojang.math.Quaternion;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.core.webapi.WebManager;
+import com.wynntils.core.webapi.profiles.MapProfile;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.render.HorizontalAlignment;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.render.VerticalAlignment;
+import com.wynntils.mc.utils.McUtils;
+import net.minecraft.client.renderer.GameRenderer;
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GL13;
+
+public class MiniMapOverlayFeature extends UserFeature {
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    public final MiniMapOverlay miniMapOverlay = new MiniMapOverlay();
+
+    public static class MiniMapOverlay extends Overlay {
+
+        @Config
+        public float scale = 1f;
+
+        @Config
+        public boolean followPlayerRotation = true;
+
+        @Config
+        public boolean renderUsingLinear = true;
+
+        @Config
+        public MapMaskType maskType = MapMaskType.Rectangular;
+
+        public MiniMapOverlay() {
+            super(
+                    new OverlayPosition(
+                            0,
+                            0,
+                            VerticalAlignment.Top,
+                            HorizontalAlignment.Left,
+                            OverlayPosition.AnchorSection.TopLeft),
+                    new GuiScaledOverlaySize(200, 200));
```

Comment:
The default value looked huge, at least on my monitor. Maybe lower it to `100, 100`, or at least `150, 150`?

Reply:
Will do :)